kernel/configs/fragments/4.19.x/common/network.conf

```diff
@@ -0,0 +1,269 @@
+# Our networking requirements
+### FIXME - this probably needs a good review ###
+
+# pre-reqs
+CONFIG_NETDEVICES=y
+CONFIG_PROC_FS=y
+CONFIG_SYSFS=y
+CONFIG_SECURITY=y
+
+# The list
+CONFIG_NET=y
+CONFIG_ETHERNET=y
+CONFIG_NET_CORE=y
+CONFIG_NET_INGRESS=y
+CONFIG_PACKET=y
+CONFIG_PACKET_DIAG=y
+CONFIG_UNIX=y
+CONFIG_XFRM=y
+CONFIG_XFRM_ALGO=y
+CONFIG_XFRM_USER=y
+CONFIG_XFRM_SUB_POLICY=y
+CONFIG_XFRM_MIGRATE=y
+CONFIG_XFRM_STATISTICS=y
```

Comment:
I think we can get rid of this, looks like a developer feature.

Reply:
thx. also dropped 'XFRM_MIGRATGE', as the Kconfig file says that is really used in situations like mobile ipv6 - which I suspect we never do.